lib/windowMaid.js

```diff
@@ -0,0 +1,86 @@
+// Third Party Modules
+const _ = require('lodash')
+
+// Logging
+const debug = require('debug')
+const logger = debug('electronpdf:')
+
+/** How long a window can remain open before it is terminated, in milliseconds */
+const HUNG_WINDOW_THRESHOLD = process.env.ELECTRONPDF_WINDOW_LIFE_THRESHOLD || 1000 * 60 * 5 /* minutes */
+
+// Window Cache - Keep track of all windows created, and if any get stuck close
+// them
+const windowCache = {}
+
+class WindowMaid {
```

Comment:
I don't see how the `static` keywords make it any more complex, but I don't mind changing it either so I'll do what you suggest.
And if it was part of a public API I think the argument about instantiation makes great sense.  Since it's not in this case I'm not super concerned about that.